### PR TITLE
Added helpful print messages to aid in debugging

### PR DESCRIPTION
**Pull request by madhavajay**
## Description
- Added icon, name and pretty print property to many things
- Added emoji converter to turn UID Hex into easily matchable Visual ID
- Fixed issues with node tests and flowing HeritageUpdateMessage back up
- Added more debugging commands to visually see whats happening
- Added lookup_id to RegisterChildNodeMessage to ensure consistent key
- Added verify_key emoji printing for easy visual matching

## Affected Dependencies
None

## How has this been tested?
Visually

## Checklist
- [x] I have followed the [Contribution Guidelines](https://github.com/OpenMined/.github/blob/master/CONTRIBUTING.md) and [Code of Conduct](https://github.com/OpenMined/.github/blob/master/CODE_OF_CONDUCT.md)
- [x] I have commented my code following the [OpenMined Styleguide](https://github.com/OpenMined/.github/blob/master/STYLEGUIDE.md)
- [x] I have labeled this PR with the relevant [Type labels](https://github.com/OpenMined/.github/labels?q=Type%3A)
- [x] My changes are covered by tests

### CODE REVIEW

**Comment by madhavajay** [comment]
I am finding it really hard to follow everything going on when debugging which results in lots of print messages to cave man debug. I honestly think it's slowing me down and giving me a physical headache. 😂 So I decided to make this less painful so I can see whats going on and follow it through.

I haven't added this everywhere yet just what was needed in tracing the node test through to see why one of them was failing.

The idea behind the Visual ID for addresses is to take the last 8 hex chars and turn them into 2 emojis. That way at a glance it's really easy to match UIDs visually.

I'm finding it really helpful but I understand if you think this is too intrusive. Still it might be good if we had a logging layer we could use and have this as a toggle.

Screen Shot 2020-08-15 at 2 17 22 pm

**Comment by madhavajay** [comment]
I fixed the node tests, but I need to stop for the night.

Screen Shot 2020-08-15 at 10 08 37 pm

**Comment by madhavajay** [comment]
Okay, so the tests pass but the code coverage is a little behind, I will fix it up later but just wanted to at least get the tests / PR stable. 😴

**Comment by iamtrask** [comment]
Phenominal contributions! I've also found this very frustrating! Thank you for making the experience better for you and for all of us :)
